arch/arm/src/stm32/Kconfig

```diff
@@ -8104,6 +8104,8 @@ config STM32_HRTIM_FAULT4
 
 endif # STM32_HRTIM_FAULTS
 
+if !STM32_STM32G47XX
```

Comment:
@daniel-p-carvalho instead of blocking it for STM32_STM32G47XX it is better to define a symbol STM32_HRTIM_HAS_CLK_PLL and enable it form STM32 families that support it

Reply:
+1 for what Alan wrote